### PR TITLE
EIP1-4186: EIP1:4352: API spec for updating AED status

### PR DESCRIPTION
**Pull request by vishalgupta-valtech**
image

**Updated to use `certificateNumber`** 
image

### CODE REVIEW

File: src/main/resources/openapi/PrintAPIs.yaml [hunk 1]
```diff
@@ -682,11 +682,115 @@ paths:
...
+  # --------------------------------------------------------------------------------
+  '/eros/{eroId}/anonymous-elector-documents/{documentNumber}/status':
+    parameters:
+      - name: eroId
+        description: The ID of the Electoral Registration Office responsible for an AED application.
+        schema:
+          type: string
+        in: path
+        required: true
+      - name: documentNumber
```
Comment (Neil-Massey) [inline src/main/resources/openapi/PrintAPIs.yaml:696]: Is this the same as the `certificate_number` in the database?

**Review by sentioservices — APPROVED** [review]
LGTM

**Review by matt-wills-valtech — CHANGES_REQUESTED** [review]
I don't think we should make a partial change to switch to `documentNumber` before we're received confirmation.

In the meantime, we could opt for `certificateNumber` and change it everywhere in the print-api if the need arises (not least because it may take a while to get a decision!), but let's decide tomorrow 👍

**Comment by vishalgupta-valtech** [comment]
> Robin has confirmed we should stick with certificate number:
> 
> image



> Robin has confirmed we should stick with certificate number:
> 
> image

Thank you @matt-wills-valtech . I have updated the PR and now using a certificateNumber in endpoint url 👍🏼 
image

**Review by matt-wills-valtech — APPROVED** [review]
LGTM

**Comment by vishalgupta-valtech** [comment]
Follow up PR that corrects a little typo https://github.com/cabinetoffice/eip-ero-print-api/pull/183

**Pull request merged**